### DISCUSSION TITLE
Hierarchical codes possible?

### DISCUSSION THREAD

**Discussion by kjoetom** [ccbogel/QualCoder#769] [Q&A] [answered]
Hi there,

I very much appreciate that you are developing this free open source software and making it available to everyone. Thank you.

I am currently trying to switch from a commercial software to QualCoder, but I am running into the following problem.

We used to use a hierarchical system of codes.

As far as I understand it correctly, codes cannot be arranged hierarchically in Qualcoder. This is apparently only possible for categories.

From this point of view, there are only two possibilities:
1. one includes the hierarchy in the names of the codes (e.g. A_B_C_D_E).
2. one creates the categories A to E, arranges them hierarchically and creates exactly 1 code with the same name within each category.

Unfortunately, both are confusing when tagging and error-prone.

Is there perhaps a way to arrange codes (not categories) hierarchically in QualCoder? Is this planned for the future?

Kind regards
kjoe

**Comment by ccbogel** [comment, marked as answer]
Hello, thanks for your interest. I based how this works originally on RQDA.

"Is there perhaps a way to arrange codes (not categories) hierarchically in QualCoder? Is this planned for the future?"
- No, not now. If I had known of this years ago, I could have initially developed the data structure to match, but now, I feel, there is far too much to change and test to meet this requirement. And there would be no compatibility with existing projects that people are using.
- 
- Your suggestions 1 and 2 are solutions that would work,
